docs/api-testing/use-cases/saving-token-kv.md

```diff
@@ -0,0 +1,278 @@
+  var: token_var
+- id: if
+  children:
+    - id: get
+      children: []
+      url: https://m2-loadtest.load2.apifortress.com/token
+      var: payload
+      mode: json
+    - id: set
+      var: token_var
+      mode: string
+      value: ${payload.token}
+    - id: kv
+      key: token
+      action: set
+      object: token_var
+  expression: "!token_var"
+- id: assert-exists
+  expression: token_var
+- id: post
+  children:
+    - id: header
+      name: Authorization
+      value: ${token_var}
+  url: https://m2-loadtest.load2.apifortress.com/profile
+  var: profilePayload
+  mode: json
+```
+
+### Using the Key/Value in Other Tests
+
+To use the Key/Value Store in other tests in the same Organization, you have to first load the value from the Key/Value Store and then use it.
+
+<img src={useBaseUrl('img/api-testing/kv-examples/token-kv-tests.png')} alt="Load the key/value in other tests" width="450"/>
+
+- Action - for example `Load`
+- Key - for example `token`
+- Variable - for example `my_token`
+
+`token` is the **Key** you assigned in the [first step](/#saving-a-token-in-a-keyvalue-store) of the previous example.
+`my_token` is the variable name you will use inside the test for referencing it.
+
+## Refreshing the Token
+
+The previous example shows how to save the token in the Key/Value Store but, once created, the token will be the same. In reality, tokens usually expires after a certain amount of time so you need to generate a new one. In the following example, you will see how to improve the previous example by refreshing the token.
+
+1. First, add the **Key/Value Store** component.
+
+   - Action - for example `Load`
+   - Key - for example `token`
+   - Variable - for example `token_var`
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/token-kv1.png')} alt="load method in key value component" />
+
+   It loads the token: the first time you run the test, the value will be empty then, it will be the token retrieved from the call.
+
+2. **Save Changes**.
+
+3. Next, add the **Key/Value Store** component.
+
+   - Action - for example `Load`
+   - Key - for example `last_token`
+   - Variable - for example `last_token_var`
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/refresh-kv1.png')} alt="load method in key value component" />
+
+   In this step, you load the date and time the token was last generated.
+
+4. **Save Changes**.
+
+5. Add the **If** component, then **Save Changes**.
+
+   - Expression - for example `!token_var || ! last_token_var || last_token_var+60000 < D.nowMillis()`
+
+   <img src={useBaseUrl('img/api-testing/kv-examples/refresh-if.png')} alt="if component" />
+
+   In this step you check if there is a token in the Key/Value store, the date is present and one minute has not passed since last token generation. If any of these conditions are not met, you generate a new one. The example refreshes the token every minute (60000ms), you can choose to refresh it differently by changing the value.
```

Comment:
```suggestion
   In this step you check if there is a token in the Key/Value store, the date is present and one minute has not passed since the last token generation. If any of these conditions are not met, you generate a new one. The example refreshes the token every minute (60000ms), you can choose to refresh it differently by changing the value.
```